### ISSUE TITLE
Deduplication of bytes-data file at the storage layer

### ISSUE THREAD

**Issue by Donnype** [minvws/nl-kat-coordination#427]

**Comment by underdarknl** [comment]
True, I think it would be useful at least to advise users on using a deduplicating, and compressing filesystem for bytes data.
Further optimization on our end could include tracking all duplicate files per hash, but I'm guessing we will be getting different hashes due to timestamps in the output streams anyway. That would result in us re-implementing block by block deduplication already present in various filesystems.
